misc/librenms-watchdog.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
This is a good compromise.

Reply:
Yeah, hopefully this should avoid any issues introducing the code.
Thanks for the review!